tests/providers/amazon/aws/hooks/test_s3.py

```diff
@@ -466,16 +466,26 @@ def test_delete_bucket_if_not_bucket_exist(self, s3_bucket):
             assert mock_hook.delete_bucket(bucket_name=s3_bucket, force_delete=True)
         assert ctx.value.response["Error"]["Code"] == "NoSuchBucket"
 
-    @mock.patch.object(S3Hook, "get_connection", return_value=Connection(schema="test_bucket"))
+    from airflow.providers.amazon.aws.utils.connection_wrapper import AwsConnectionWrapper
```

Comment:
Import should be defined in top level, unless we have strong reason import here

Reply:
Good catch, sorry about that. I have removed this.